src/libsync/theme.cpp

```diff
@@ -284,7 +284,7 @@ QString Theme::defaultServerFolder() const
 
 QString Theme::helpUrl() const
 {
-    return QStringLiteral("https://doc.owncloud.org/desktop/%1.%2/").arg(OCC::Version::version().majorVersion()).arg(OCC::Version::version().microVersion());
+    return QStringLiteral("https://doc.owncloud.com/desktop/%1.%2/").arg(OCC::Version::version().majorVersion()).arg(OCC::Version::version().microVersion());
```

Comment:
Why not writing: `https://doc.owncloud.com/desktop/latest/`
This would resolve in any case to a valid address.
Imagine one keeps the app longer than the actual 2 releases. Such a case would end up in a 404 while using latest would always resolve correct.
This is also true when using `https://doc.owncloud.com/desktop/latest/installing.html` which would finally reslove to `https://doc.owncloud.com/desktop/3.2/installing.html`

Reply:
The intention here is clear, if you have a 3 client, don't show the 4 doc.
I guess a better solution would be to not serve a 404 but redirect to latest.